proto/stat.proto

```diff
@@ -58,6 +58,17 @@ message SourceSeries {
   string provenance_url = 9;
 }
 
+message Series {
+  // Map from date to stat value.
+  map<string, double> val = 1;
+  // Series metadata.
+  StatMetadata metadata = 2;
+}
+
+message SeriesMap {
+  // Map from key to series. The key could be place dcid or stat var dcid.
```

Comment:
Depending on...?

Reply:
Updates the comment to make it more generic. What is key is really depending on the response structure so not worth emphasize in here.